src/components/TodoList.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState, useEffect } from "react";
+import TodoItem from "./TodoItem";
+import { useRef } from "react";
```

Comment:
We can merge related imports into a single statement here 
```suggestion
import { useState, useEffect, useRef } from "react";
import TodoItem from "./TodoItem";
```